core/src/main/java/com/linecorp/armeria/client/HttpChannelPool.java

```diff
@@ -598,10 +595,6 @@ public void release() {
                 addToPool(protocol(), key, this);
             } else {
                 // Channel not healthy, just releasing it.
```

Comment:
Could modify the comment as well because there's no releasing code in this `else` block?